packages/avatars/src/styled/StyledStandaloneStatusIndicator.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+import { math } from 'polished';
+
+import { IStatusIndicatorProps, STATUS } from '../types';
+import { getStatusColor, statusIconStyles, TRANSITION_DURATION } from './utility';
+
+const COMPONENT_ID = 'avatars.status-indicator.indicator';
+
+const [available, away, transfers, offline] = STATUS;
+
+const sizeStyles = (props: IStatusIndicatorProps & ThemeProps<DefaultTheme>) => {
+  const borderWidth = props.theme.shadowWidths.sm;
+
+  let size = '0';
+  let marginTop = props.theme.space.base;
+
+  if (props.isCompact) {
+    marginTop = props.theme.space.base * 1.5;
+    size = math(`${props.theme.space.base * 3}px - (${borderWidth} * 2)`);
+  } else {
+    size = math(`${props.theme.space.base * 4}px - (${borderWidth} * 2)`);
+  }
```

Comment:
Are we setting up precision layout magic that might better be handled via flex?

Reply:
This would require an additional element when considering how the indicator will line up when text is overflowing. Currently the containing parent `figure` element does use `inline-flex` which lays out the indicator and caption.